src/store/Business/state.ts

```diff
@@ -0,0 +1,18 @@
+import { BusinessStateIF } from '@/interfaces'
+
+export const state: BusinessStateIF = {
+  businessInfo: {}, // *** TODO: populate defaults as below
```

Comment:
I'm trying to figure out what to do here (instead of needing to commit all the individual properties). But it looks funny having this business module with just 1 property (businessInfo).